app/services/instance_config_service.py

```
@@ -200,6 +200,141 @@ def set_smtp_config(
+    def is_oidc_env_configured(self) -> bool:
+        return (
+            settings.oidc_enabled is not None
+            or settings.oidc_issuer is not None
+            or settings.oidc_client_id is not None
+            or settings.oidc_client_secret is not None
+        )
+
+    def set_oidc_config(
+        self,
+        db: Session,
+        oidc_enabled: bool,
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)